contracts/Rentals.sol

```diff
@@ -177,7 +179,7 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver {
         uint256 _index,
         uint256 _rentalDays,
         bytes32 _fingerprint
-    ) external {
```

Comment:
Why not use `nonReentrant` at `_rent` directly?

Reply:
because if it reenters through acceptListing or acceptOffer, it will do some stuff before reverting when it reaches _rent

Reply:
It can't do anything because there are no interactions (at least no non static ones). But still, it's better to revert as soon as possible.